### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by stilor**

### CODE REVIEW

File: scripts/functions [hunk 1]
```diff
@@ -1786,7 +1786,7 @@ CT_Mirrors()
             '')
                 # Ignore, this happens before .config is fully evaluated
                 ;;
-            [34].*)
+            [345].*)
```
Comment (cpackham) [inline scripts/functions]: Is it worth making this `[3-9].*` just to be future proof?

**Comment by cpackham** [comment]
Had a glance over the series. Looks good to me. I'll do a few tests builds.

**Comment by stilor** [comment]
Yes, thanks. I saw that but the change I had in my tree has added 5.3.x as well - which has since been released.

**Comment by stilor** [comment]
Known breakage is that newer gdbserver fails to build against uClibc-ng due to `in6_addr` not visible. Looking at it right now.

**Comment by cpackham** [comment]
Running:
- [ ] arm-unknown-linux-gnueabi
- [ ] powerpc64-unknown-linux-gnu
- [ ] x86_64-unknown-linux-gnu

**Pull request merged**